nextflow.config

```diff
@@ -12,6 +12,7 @@
 params {
     // Input options
     input = null
+    sample_limit = 500
```

Comment:
Instead of `sample_limit` could you name it `max_samples` so that it matches the other parameters like `max_cpus` or `max_memory`?